posthog/models/channel_type/sql.py

```diff
@@ -0,0 +1,59 @@
+import json
+import os
+
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+CHANNEL_DEFINITION_TABLE_SQL = f"""
+CREATE TABLE IF NOT EXISTS channel_definition ON CLUSTER {CLICKHOUSE_CLUSTER} (
+    domain String NOT NULL,
+    kind String NOT NULL,
+    domain_type String NULL,
+    type_if_paid String NULL,
+    type_if_organic String NULL,
+) ENGINE = MergeTree()
```

Comment:
This table should probably be `ReplicatedMergeTree` right? We don't ensure that migrations are run directly on every node (why we use `ON CLUSTER` to have the coordinating node pass the DDLs onto each member of the cluster) so when this migration runs you will only load the data onto one node.